### ISSUE TITLE
Number of Referrals Made page

### ISSUE THREAD

**Issue by jkondrat** [benetech/ServiceNet#1237]
Scope:
* A separate tab on Referral history page including two paginated tables: Referrals made to, referrals made from
* filtering the records

**Comment by jkondrat** [comment]
![Screenshot from 2020-09-17 12-23-07](https://user-images.githubusercontent.com/5969801/93458529-b4ecc500-f8e0-11ea-9e1f-84d8ca725247.png)

**Comment by jkondrat** [comment]
@johnhbenetech @EnriqueCorado we have a couple of questions about the second table:
* Shouldn't it show number of referrals made from an organization instead of a list of all referrals with statuses?
* What does the number in the first column stand for? Is this a phone number/id?

**Comment by EnriqueCorado** [comment]
@jkondrat 

- Correct, second column # of referrals made from an organization

- first column, the organization where referrals are coming from 

I will add a new issue to the product board to potentially add a new tab with the referral status of all referrals. Basically, a table showing weekly/monthly numbers of beneficiaries and their status to potentially gauge how much food, for example, they need for a specific date.

**Comment by awalkowiak** [comment]
@EnriqueCorado I have two questions about these tables:
1. Left-hand table: Should we show the count of referrals made only from the current user's organizations or made from organizations of all users in his user group?
2. Right-hand table: Should we show referrals made only to the current user's organizations or to all organizations from his user group?

**Comment by EnriqueCorado** [comment]
@awalkowiak 

1. Let's start with the count of referrals made only form the current user's organization 
2. Let's show referral made only to the current user's organizations

**Comment by awalkowiak** [comment]
@EnriqueCorado sounds good, thanks.

**Comment by abaranowska1** [comment]
@awalkowiak 
It looks like Organizations with Waiting for Arrival status are not displayed on the Number of Referrals page and CSV file.
 
![image](https://user-images.githubusercontent.com/61964176/97006563-57cbdb00-1540-11eb-9954-2b0c9cab4a71.png)
![image](https://user-images.githubusercontent.com/61964176/97006573-5ac6cb80-1540-11eb-8a45-c8bcf2d2b349.png)

**Comment by awalkowiak** [comment]
@abaranowska1 everything seems to be working for me. You could't see any referrals made to with 'waiting for arrival' status for this user's orgs because there were no waiting referrals to this user. I went to 'bognaprov' user and I made a referral to org from 'provider' user's organization and now there is 'Waiting for arrival' record. Seems to be working.

**Comment by abaranowska1** [comment]
@awalkowiak 
I confirmed that I can see also Organizations with 'waiting for arrival' status, I picked the wrong Organization. Thanks for the explanation of this task. 
![image](https://user-images.githubusercontent.com/61964176/97178089-50ead580-1797-11eb-8171-27b6eccab03a.png)

**Tested with success**
- Number of Referrals Made page was created. Filters are working fine and Organizations are displayed correctly. 
- It is possible to download the CSV files
